### PR TITLE
increasing the time limit tolerance to running the test

### PR DESCRIPTION
**Pull request by odorifqi**
as the suggestion of the author, i increased the time limit max to 10x (590), so when the test running it would accept wider range of test time in case of the test consuming longer time.

### CODE REVIEW

File: test/helper.spec.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ describe('runner', () => {
             await templ(bigodon);
             const deltaT = Date.now() - start;
             expect(deltaT).to.be.at.least(49);
```
Comment (gabriel-pinheiro) [inline test/helper.spec.js:66]: I would be better to change this and the two wait helpers above to 10x too (this 49 becomes 490, `{{wait 20}}` becomes `{{wait 200}}`, and so on). That way, the limit will also fail for longer than expected waits

**Review by gabriel-pinheiro — CHANGES_REQUESTED** [review]

File: test/helper.spec.js (hunk 1 above)
Comment (odorifqi) [reply test/helper.spec.js]: I'm sorry for that, i have change it. Thanks for the feedback 😄

**Review by gabriel-pinheiro — APPROVED** [review]

**Comment by gabriel-pinheiro** [comment]
Looks good to me! Thanks for your contribution 😄

**Comment by odorifqi** [comment]
Thanks a lot for helping me do my first ever PR 😄

**Pull request merged**